### ISSUE TITLE
Migration from v6 to v7: mention `includePaths` → `loadPaths`

### ISSUE THREAD

**Issue by julien-deramond** [oddbird/true#247]
Hey there :)

This is not really an issue but rather a feedback after a migration from v6 to v7.

We had something like that:

```js
  runSass({
    data: sassString,
    includePaths: [path.dirname(filename)]
  }, { describe, it })
```

that became:

```js
  runSass(
    { describe, it, sourceType: 'string' },
    sassString,
    { loadPaths: [path.dirname(filename)] }
  )
```

I followed the migration guide in [v7.0.0 - Migrating from v6](https://github.com/oddbird/true/releases/tag/v7.0.0) (thx for that! it's already nice to have one).

IDK if it's used by a lot of folks but I'd suggest to maybe add something that mentions the `includePaths` → `loadPaths` modification. Maybe it's mentioned somewhere else but I've only found out how to migrate it by looking at your source code (via https://github.com/oddbird/true/commit/cda74efb605a2a1c76323d5ee566649d056be898).

**Comment by jgerigmeyer** [comment]
@julien-deramond Fixed in #248 -- let me know what you think!

**Comment by julien-deramond** [comment]
Thanks a lot or the update @jgerigmeyer, it'll be helpful for folks!
